wikimedia/wikipedia-ios
# [ALT TEXT] Alt text trigger panel

**Phabricator:** https://phabricator.wikimedia.org/T370229

### Notes
* Some of the verification logic in this PR depends on https://github.com/wikimedia/wikipedia-ios/pull/4891, so it'll need to be updated after it is merged.
* But some of the other logic can be verified and reviewed before that if necessary
* This PR adds some verifications to trigger the Alt Text task panel
* A new panel was added according to the designs

### Test Steps
1. With the dev settings toggle on, go into image recommendations on a target language
2. Add a recommendation, add alt text
3. You should not see the panel
4. Add another recommendation, **do not add** alt text
6. After publishing, you should see the panel
7. Note that the panel has no actions, and the bottom sheet is not shown underneath (the bottom sheet can't push/present). Bringing it back on after hitting "do not add" should be done with the survey task
8. Go back to the home page, enter image recs again
9. Add a recommendation without alt text
10. You should not see the panel

### Screenshots/Videos
